spec/models/search/bookmark_search_form_spec.rb

```diff
@@ -215,6 +215,75 @@
     end
   end
 
+  describe "search_results" do
```

Comment:
Sorry, I should have made it clear that I meant for the describe blocks to use the name of the methods being tested. So the bookmarkable specs (calling `bookmarkable_search_results`) were fine where they were, only the bookmark specs (calling `search_results` should be moved to the new block.

Reply:
Nope, that's me who can't get a grasp on it 😄 
Thank you, done.